### ISSUE TITLE
How to Create Native Android APP

### ISSUE THREAD

**Issue by Pengran** [anismiles/websocket-android-phonegap#9]
Hi,

I have read the README and tried to create a native Java app in Android. It writes following in the README: 
Create a class, say WebSocket extending WebSocketListener

But I don't find WebSocketListener in the package. Please kindly give me some help about this problem.

/Pengran

**Comment by anismiles** [comment]
Dude. It was a typo. It should have been "WebSocketListener extending WebSocket" hope this resolves your issue. 
- Animesh

**Comment by zidagar** [comment]
Ok, I've extended the WebSocket class and now? How I should do? (have you got a simple example with webview?)
Thanks

**Comment by anismiles** [comment]
No. What exactly you are looking for? A native websocket implementation for android? If that's the case, then you can use any websocket client and wrap that up in a service.

**Comment by fkaking** [comment]
how  do i use it in a native Android APP with a common webview?
now i have extended WebSocket,and then?
